java-manta-it/src/test/java/com/joyent/manta/client/MantaClientSigningIT.java

```diff
@@ -225,8 +221,11 @@ public final void testCanCreateSignedOPTIONSUriFromPath()
         }
     }
 
-    @Test
     public final void testCanCreateSignedURIWithEncodedCharacters() throws IOException {
+        if (config.isClientEncryptionEnabled()) {
+            throw new SkipException("Signed URLs are not decrypted by the client");
```

Comment:
I'm not sure I'm following what signed urls have to do with client side encryption.

Reply:
They aren't. It is a spurious test failure that I discovered when rerunning many of the CSE tests. The signed URL tests were never intended to be used with CSE.